### ISSUE TITLE
Invalid config "Enabling the target status feature"

### ISSUE THREAD

**Issue by crwatanabe** [GoogleCloudPlatform/prometheus-engine#507]
I have added the following statement according to this section. This is an error.
https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#target-status

```diff
# Please edit the object below. Lines beginning with a '#' will be ignored,
# and an empty file will abort the edit. If an error occurs while saving this file will be
# reopened with the relevant failures.
#
# operatorconfigs.monitoring.googleapis.com "config" was not valid:
# * <nil>: Invalid value: "The edited file failed validation": ValidationError(OperatorConfig): unknown field "features" in com.googleapis.monitoring.v1.OperatorConfig
#
apiVersion: monitoring.googleapis.com/v1
kind: OperatorConfig
managedAlertmanager:
  configSecret:
    key: alertmanager.yaml
    name: alertmanager
+ features:
+   targetStatus:
+     enabled: true
metadata:
  annotations:
    components.gke.io/layer: addon
  creationTimestamp: "2023-06-28T05:51:38Z"
  generation: 1
  labels:
    addonmanager.kubernetes.io/mode: EnsureExists
  name: config
  namespace: gmp-public
```

```sh
$ kubectl -n gmp-public edit operatorconfig config
error: operatorconfigs.monitoring.googleapis.com "config" is invalid
A copy of your changes has been stored to "/var/folders/p2/xxxxxxxxxxxx/T/kubectl-edit-xxxxxxxx.yaml"
error: Edit cancelled, no valid changes were saved.
```

**Comment by crwatanabe** [comment]
Maybe the version I'm using now is out of date, 1.24.12-gke.500.

**Comment by lyanco** [comment]
Yep - this feature is only available on 1.25+.
